src/VisualizationTests/WpfVisualizationTests.csproj

```diff
@@ -16,7 +16,7 @@
     <PackageReference Include="HelixToolkit" Version="2.17.0" />
     <PackageReference Include="HelixToolkit.Wpf" Version="2.17.0" />
     <PackageReference Include="HelixToolkit.Wpf.SharpDX" Version="2.17.0" />
-    <PackageReference Include="Moq" Version="4.2.1507.0118" />
+    <PackageReference Include="Moq" Version="4.18.4" ExcludeAssets="none" />
```

Comment:
moq should be excluded here.
I think only DynamoCoreTests should have ExcludeAssets="none"

Reply:
@pinzart90 can you explain why? I was thinking it would be simpler if these package references all looked the same when possible but open to other approaches for sure.

Do you think it's easier to control if only a single reference brings in the binaries so it's easy to see what project is bringing them in?

Reply:
well... that sounds reasonable, especially if we will move towards a solution like that. 
I'm not sure it will help for nunit framework / ms test.sdk or the dependencies we are already stuck with in the bin folder - but I will follow that advice for now and only copy from DynamoCore to make it simpler to undo later.

Reply:
So I've done this - kind of - now actually none of the moq package references are `excludeAssets="none"` - instead I'm doing manual copy steps as I don't want to pull the transitive deps into the bin folder.

This is kind of a bummer, I wonder if long term it might make sense to setup a different .sln for test projects - OR just set test projects to have a completely different output path.

Reply:
something I hope we can look at in:
https://jira.autodesk.com/browse/DYN-5673
FYI @jasonstratton